.circleci/config.yml

```
@@ -316,3 +357,33 @@ workflows:
     - ipfs-webui:
         requires:
           - build
+    - docker-build
+    - docker-push:
+        requires:
+        - docker-build
+        - golint
+        - gotest
+        - sharness
+        - interop
+        - go-ipfs-api
+        - go-ipfs-http-client
+        - ipfs-webui
+
+  # NOTE: You have to opt in to get circleci to trigger builds for a tat. As it is required that you set the tag filter to all dependent jobs, this is pulled out into a seperate workflow.
```

Typo ;)

```suggestion
  # NOTE: You have to opt in to get circleci to trigger builds for a tag. As it is required that you set the tag filter to all dependent jobs, this is pulled out into a seperate workflow.
```

But I also had to read this twice to make sure I understood. Maybe this is clearer?

```suggestion
  # NOTE: CircleCI only builds tags if you explicitly filter for them. That
  # also means tag-based jobs can only depend on other tag-based jobs, so we
  # use a separate workflow because every job needs to be tagged together.
```